core/store/migrations/migration1568833756/migrate.go

```diff
@@ -0,0 +1,45 @@
+package migration1568833756
+
+import (
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/jinzhu/gorm"
+	"github.com/pkg/errors"
+	"github.com/smartcontractkit/chainlink/core/store/models"
+	"gopkg.in/guregu/null.v3"
+)
+
+// InitiatorParams is a collection of the possible parameters that different
+// Initiators may require.
+type InitiatorParams struct {
```

Comment:
Does this need to be declared here? I don't think it's really being used, you can just use a string below on line 34.